src/handlers/handlePreviousQuestion.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+import showCurrentQuestion from "./showCurrentQuestion.js";
+import { quizData } from '../data.js';
+
+const  handlePreviousQuestion = () => {
+    if (quizData.currentQuestionIndex < 1) {
+        quizData.questions[0];
```

Comment:
What does this do?

Reply:
This prevents the previous button from going back when it is in the first question.